### ISSUE TITLE
ImportError: cannot import name 'nms_rotated_ext' from partially initialized module 'utils.nms_rotated' (most likely due to a circular import) 

### ISSUE THREAD

**Issue by AhmedSharba** [hukaixuan19970627/yolov5_obb#642]
how can I solve this error

**Comment by veronicamorelli** [comment]
I also have the same error, I don't know where nms_rotated_ext is defined. Did you solve it?

**Comment by AhmedSharba** [comment]
No, I tried to solve but I can’t, if you solve it please inform me because
I need it necessarily.

With regards

**Comment by bluceliuljx** [comment]
i have the same issue

![image](https://github.com/hukaixuan19970627/yolov5_obb/assets/40584477/04ae480a-7375-4271-aff5-845db52e61fd)

**Comment by thalia0503** [comment]
run:
cd utils/nms_rotated
python setup.py develop


to get file nms_rotated_ext.cpython-38-x86_64-linux-gnu.so

**Comment by AhmedSharba** [comment]
Thank you for reply but the link doesn't work

**Comment by charlesjcliffe** [comment]
Got the same error when I added yolov5_obb dir to the python path; just change the import at the top of nms_rotated_wrapper -- 

Replace: 

`from . import nms_rotated_ext`

with

`import nms_rotated_ext`

and it'll resolve the circular import.
